data/tribes/initialization/atlanteans/units.lua

```diff
@@ -1473,45 +1513,80 @@ descriptions:new_tribe {
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Hunter's House
             purpose = pgettext("building", "Hunts animals to produce meat."),
             -- TRANSLATORS: Note helptext for an atlantean production site: Hunter's House
-            note = pgettext("atlanteans_building", "The hunter’s house needs animals to hunt within the work area.")
+            note = pgettext("atlanteans_building", "The hunter’s house needs animals to hunt within the work area."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site: Hunter's House
+            lore = pgettext("atlanteans_building", "Sometimes we end up in a region with no fish, so we must hunt, "..
+                                                   "Game will reproduce by itself, if not hunted too much. But yes, fish is our main meal."),
```

Comment:
```suggestion
            lore = pgettext("atlanteans_building", "Sometimes we end up in a region with no fish, so we must hunt, "..
                                                   "game will reproduce by itself, if not hunted too much. But yes, fish is our main meal."),
```
No capital Letters in the Middle of a Sentence

Reply:
I think it should be a new sentence instead.

Reply:
Made it a new sentence